CreationService/CommandRouter.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Threading.Tasks;
+using CreationService.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CreationService
+{
+    internal static class CommandRouter
+    {
+        internal static void Route(string message)
+        {
+            JObject receivedObj = JsonConvert.DeserializeObject<JObject>(message);
+            string command = receivedObj["command"].Value<string>();
+
+            switch (command)
+            {
+                case "add":
+                    int eventId = receivedObj["EventId"].Value<int>();
+                    string location = receivedObj["Location"].Value<string>();
+                    string driveFrom = receivedObj["DriveFrom"].Value<string>();
+                    DateTime dateFrom = receivedObj["DateFrom"].Value<DateTime>();
+                    DateTime dateTo = receivedObj["DateTo"].Value<DateTime>();
+
+                    Event eventObj = new Event()
+                    {
+                        EventId = eventId,
+                        Location = location,
+                        DriveFrom = driveFrom,
+                        DateFrom = dateFrom,
+                        DateTo = dateTo
+                    };
+
+                    EventRequest r = new EventRequest(Guid.NewGuid(), eventObj);
+                    EventRequestRepository.Instance.AddRequest(r);
+
+                    JObject requestCar = new JObject();
+                    requestCar["from"] = dateFrom;
+                    requestCar["to"] = dateTo;
+                    requestCar["stationId"] = driveFrom;
```

Comment:
Wouldn't it be better to name it `station` as we are passing the station name?

Reply:
The legacy system requires a string as an ID,l even though on the tests it is written "Porto", we might use something different such as "23".